Description: Upstream changes introduced in version 1.4.0-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 google-mock (1.4.0-2) unstable; urgency=low
 .
   * Depend on libgtest instead of building internal version (Closes: #580770)
 .
 The person named in the Author field signed this changelog entry.
Author: Fredrik Hallenberg <hallon@debian.org>
Bug-Debian: http://bugs.debian.org/580770

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- google-mock-1.4.0.orig/src/gmock-spec-builders.cc
+++ google-mock-1.4.0/src/gmock-spec-builders.cc
@@ -52,7 +52,7 @@ namespace internal {
 
 // Protects the mock object registry (in class Mock), all function
 // mockers, and all expectations.
-Mutex g_gmock_mutex(Mutex::NO_CONSTRUCTOR_NEEDED_FOR_STATIC_MUTEX);
+Mutex g_gmock_mutex;
 
 // Constructs an ExpectationBase object.
 ExpectationBase::ExpectationBase(const char* file, int line)
--- google-mock-1.4.0.orig/src/gmock-internal-utils.cc
+++ google-mock-1.4.0/src/gmock-internal-utils.cc
@@ -103,7 +103,7 @@ FailureReporterInterface* GetFailureRepo
 }
 
 // Protects global resources (stdout in particular) used by Log().
-static Mutex g_log_mutex(Mutex::NO_CONSTRUCTOR_NEEDED_FOR_STATIC_MUTEX);
+static Mutex g_log_mutex;
 
 // Returns true iff a log with the given severity is visible according
 // to the --gmock_verbose flag.
